src/ViewRenderer.php

```diff
@@ -37,51 +43,61 @@ final class ViewRenderer implements ViewContextInterface
     private Aliases $aliases;
     private WebView $view;
 
-    private string $viewBasePath;
+    private string $viewPath;
     private ?string $layout;
+    private ?string $name = null;
 
     /**
      * @var object[]
      */
     private array $injections;
 
-    private ?string $name = null;
-    private ?string $viewPath = null;
-
     /**
-     * @param DataResponseFactoryInterface $responseFactory
-     * @param Aliases $aliases
-     * @param WebView $view
-     * @param string $viewBasePath
-     * @param string|null $layout
-     * @param object[] $injections
+     * @param DataResponseFactoryInterface $responseFactory The data response factory instance.
+     * @param Aliases $aliases The aliases instance.
+     * @param WebView $view The web view instance.
+     * @param string $viewPath The full path to the directory of views or its alias.
+     * @param string|null $layout The layout name (e.g. "layout/main") to be applied to views.
+     * If null, the layout will not be applied.
+     * @param object[] $injections The injection instances.
      */
     public function __construct(
         DataResponseFactoryInterface $responseFactory,
         Aliases $aliases,
         WebView $view,
-        string $viewBasePath,
+        string $viewPath,
         ?string $layout = null,
         array $injections = []
     ) {
         $this->responseFactory = $responseFactory;
         $this->aliases = $aliases;
         $this->view = $view;
-
-        $this->viewBasePath = $viewBasePath;
+        $this->viewPath = rtrim($viewPath, '/');
         $this->layout = $layout;
         $this->injections = $injections;
     }
 
+    /**
+     * Returns a path to the directory of views that is a prefix to the relative view name.
+     *
+     * If a controller name has been set {@see withController(), withControllerName()}, it will be appended to the path.
```

Comment:
Prepended?

Reply:
No, that's right. The controller name is appended to the views directory.